tests/pricing_models/test_get_max_long.py

```diff
@@ -0,0 +1,187 @@
+"""
+Testing for the get_max_long function of the pricing models.
+"""
+
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=too-many-locals
+# pylint: disable=attribute-defined-outside-init
+# pylint: disable=duplicate-code
+
+from dataclasses import dataclass
+import unittest
+from elfpy.pricing_models.yieldspace import YieldSpacePricingModel
+
+from elfpy.types import MarketDeltas, MarketState, Quantity, StretchedTime, TokenType
+from elfpy.pricing_models.base import PricingModel
+from elfpy.pricing_models.hyperdrive import HyperdrivePricingModel
+
+
+@dataclass
+class TestCaseGetMaxLongCase:
+    """Dataclass for get_max_long test cases"""
+
+    market_state: MarketState
+    fee_percent: float
+    time_remaining: StretchedTime
+
+    __test__ = False  # pytest: don't test this class
+
+
+class TestGetMaxLong(unittest.TestCase):
```

Comment:
It's not clear to me that this is actually testing that the method found the maximum long. It seems to only be testing that the solution is a valid trade. Is that correct?